ERCS/erc-7764.md

```diff
@@ -0,0 +1,136 @@
+---
+eip: 7764
+title: Buyer-Seller Negotiable Pricing
+description: Allows buyers and sellers to freely negotiate and determine prices.
+author: S7iter (@admi-n)
+discussions-to: https://ethereum-magicians.org/t/erc-7764-buyer-seller-negotiable-pricing/20973
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-09-05
+---
+
+## Abstract
+
+This proposal introduces a new smart contract mechanism that allows buyers and sellers to freely negotiate and determine transaction prices on the Ethereum network. A new trading mode is added, where goods can be negotiated rather than only priced. This mechanism allows the seller to set an initial price, and the buyer can propose a new price through negotiation with the seller. Ultimately, both parties can reach an agreement and complete the transaction.
```

Comment:
The use of "transaction" here is somewhat ambiguous with an "Ethereum transaction." You should define what you mean by transaction here in your abstract.